teuthology/__init__.py

```diff
@@ -38,6 +48,8 @@
     format='%(asctime)s.%(msecs)03d %(levelname)s:%(name)s:%(message)s')
 log = logging.getLogger(__name__)
 
+log.info('teuthology version: %s', __version__)
```

Comment:
I don't have strong enough feelings to want to get too fancy here, so maybe using using ``log.debug`` is the right thing. If we later on feel like we really wish we had the version logged more frequently, we can revisit